frontend/.eslintrc.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
     // Sometime we need to specify any type, so it's rather useful
     "@typescript-eslint/no-explicit-any": 0,
 
+    "react/jsx-closing-bracket-location": 0,
+    "react/no-array-index-key": 0,
```

Comment:
https://stackoverflow.com/questions/46735483/error-do-not-use-array-index-in-keys \
Please do not remove it

Reply:
what you suggest using? Key is a pain in the ass.

Reply:
If it is just a dummy component you can use `index.toString()`\
What index we will use later? Maybe generate indexes on backend or you can also generate it on frontend. The problem is that if we will do some changes in this table or somehow filter/sort rows we will see how it doesn't work)